static/js/biomedical/disease/data_processing_utils.ts

```diff
@@ -285,3 +286,34 @@ export function getCompoundDiseaseContraindication(
   processedData.sort((a, b) => (a.drugSource > b.drugSource ? 1 : -1));
   return processedData;
 }
+
+/**
+ * Fetches the common name of the disease of interest
+ * @param data
+ * @returns - string with disease common name
+ */
+export function getDiseaseCommonName(data: GraphNodes): string {
+  let commonName = null;
+  if (!data) {
+    return;
+  }
+  // check for null values
+  if (_.isEmpty(data.nodes) || _.isEmpty(data.nodes[0].neighbors)) {
+    return;
+  }
+  for (const neighbour of data.nodes[0].neighbors) {
+    if (neighbour.property !== "commonName") {
+      continue;
+    }
+    // check for null or non-existent property values
+    if (_.isEmpty(neighbour.nodes)) {
+      continue;
+    }
+    commonName = neighbour.nodes[0].value;
+    // capitalize the first letter of the disease name
+    const formattedDiseaseName =
```

Comment:
Before performing these string operations, first check that there's a valid string and has length > 2.

It's safer to use `charAt` then strict array dereference. And it should start with `.charAt(0)` and `.slice(1)`

Reply:
I believe the validity of the string is checked in the `_.isEmpty()` function. I added another check for string length and if it is less than 2, then we use the DCID instead!